pkgs/tools/networking/netmeta/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "netmeta";
+  version = "unstable-2023-06-07";
+
+  src = fetchFromGitHub {
+    owner = "monogon-dev";
+    repo = "NetMeta";
+    rev = "af1f83e25e33c10fc6889c80d52b142a32d5521a";
+    hash = "sha256-T4OxUGgTbWWIo8W/4zZFHT415zR6KfDj3/fMitU19hc=";
+  };
+
+  subPackages = [
+    "cmd/helloworld"
```

Comment:
are you sure the helloworld package is needed? From what I can tell it only outputs a binary that repeatedly the current date/time and `I bims 1 container`